### ISSUE TITLE
Inspector throws an error when opening it on a nullish value

### ISSUE THREAD

**Issue by rksm** [LivelyKernel/lively.morphic#135]
```
ticking.js:59 Error executing script Script(<Inspector - inspector>>>refreshProperties()): TypeError: Cannot read property 'env' of null
TypeError: Cannot read property 'env' of null
    at Inspector_initialize_.refreshProperties (inspector.js:743)
    at TargetScript_execute_ (ticking.js:131)
    at TargetScript.Script_tick_ (ticking.js:57)
```

**Comment by merryman** [comment]
Yes, it now expects any inspected thing to be a morph. Not good. Will fix that.

**Comment by merryman** [comment]
fixed in: a8acd62
